docs/getting-started.md

```diff
@@ -282,6 +282,30 @@ If you are not able to send custom headers in your handshake request, send it as
 query parameter `authorization`. Then your path should look like this
 `/channels/<channel_id>/messages?authorization=<thing_auth_key>`.
 
+If you are using the docker environment prepend the url with `ws`. So for example
+`/ws/channels/<channel_id>/messages?authorization=<thing_auth_key>`
+
+#### Basic nodejs example
+
+```javascript
+const WebSocket = require('ws');
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"
+
+const ws = new WebSocket('wss://localhost/ws/channels/1/messages?authorization=cbf02d60-72f2-4180-9f82-2c957db929d1')
```

Comment:
Move URL in another `var`, and move API key in yet another `var`. This will make it easier for user to replace key and URL if needed.

Reply:
Is just a quick example, not real code. So keeping it small and simple could be a good thing I think

Reply:
I agree with @ferama here.

Reply:
OK, merging then.